targettests/SeparateCompilation/emulate.cpp

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// clang-format off
+// RUN: split-file %s %t && \
+// RUN: nvq++ -target quantinuum -emulate -fno-set-target-backend -c %t/emulib.cpp -o %t/emulibx.o && \
+// RUN: nvq++ -target quantinuum -emulate -c %t/emuuser.cpp -o %t/emuuserx.o && \
+// RUN: nvq++ -target quantinuum -emulate %t/emulibx.o %t/emuuserx.o -o %t/emux.a.out && \
+// RUN: %t/emux.a.out | FileCheck %s
+// clang-format on
+
+// Catch-22. We need to use argument synthesis, but if I enable that in the
```

Comment:
Is this comment old? I don't see an XFAIL comment, does the test pass with -emulate?

Reply:
Yes and yes. I'll remove this. I got the JIT parts all working here now.